src/Controller/StripeWebhookEndpoint.php

```diff
@@ -164,6 +168,10 @@ protected function handleStripeWebhook($payload, string $signatureHeader, bool $
             return new Response('Invalid signature', Response::HTTP_BAD_REQUEST);
         }
 
+        if ($this->ignores($event['type'])) {
+            return new Response('Ignored event', Response::HTTP_OK);
```

Comment:
Since it's not just ignored but deprecated, we could add a different message while keeping HTTP_OK code, e.g. "This event type 'event_name' is no longer required and can be removed in the operator webhook settings.".